keps/sig-network/2595-expanded-dns-config/README.md

```diff
@@ -185,9 +221,15 @@ the expanded DNS configuration.
 
 Yes, the feature can be disabled by disabling the feature gate.
 
-Once the feature is disabled, kube-apiserver will reject the pod having expanded
-DNS configuration and kubelet will create a resolver configuration excluding the
-overage.
+Before disabling the feature gate, is is recommended to remove pods or objects
```

Comment:
How are these identified? List all pods and check the length of the DNS search list? That sounds a little painful. Can you provide a command here so users can easily find these?

Reply:
added a command to find them.

Reply:
Nice, thanks. Yes, not simple - great to have it here so people don't have to figure it out themselves.